javascript-data-model/main.js

```diff
@@ -0,0 +1,104 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
```

Comment:
there is no need for another object here, you can put the order objects directly into the array:

```javascript
var orderHistory = [ {}, {}, {}]
```